dev-docs/RFCs/v6.1/transform-webgl1-support.md

```diff
@@ -0,0 +1,59 @@
+# RFC: WebGL1 support for Trasnform.
+
+* **Author**: Ravi Akkenapally
+* **Date**: Aug, 2018
+* **Status**: **Review**
+
+Notes:
+* PR: TODO
+
+
+## Overview
+
+With the introduction of `Transform` class, luma.gl provided an easy to use API around WebGL2's complicated TransformFeedback API and also the ability to manage/create several other WebGL resources internally. This allows applications to offload complex operation to GPU and run in parallel while making CPU available for other tasks. But this class is limited to WebGL2 context and cannot be used with WebGL1 context. In this RFC we propose an implementation to support this class for WebGL1 context.
+
+## Main idea
+
+We should be able to setup a render pipeline (strictly using WebGL1 features), such that input parameters are provided as attributes to the VertexShader, inside VertexShader user provided operations are performed, and result is passed to FragmentShader, which writes the result to a pixel as fragment color.
+
+## Implementation details
+
+### Render target
+
+Size of the target should be such that it has one pixel per each expected output. I.e. if user wants to perform an operation on two float array of size 10K , then render target should have at least 10K pixels. Type of render target :
+
+#### Option#1 Framebuffer
```

Comment:
Once we get past the initial discussion, better to offer concrete "proposals: ..." for the reader to critique rather than providing "options..." for each reader to choose from.

Reply:
I am positive on Option#1, but need to confirm with some sort of working model, hence included both options, and discussed differences.